### PR TITLE
763 sort replicasets

### LINKED ISSUE #763: UI: Sort replicasets
By the `(replicaset.name, replicaset.servers[1].name)`.

### PR DESCRIPTION
**Pull request by HustonMmmavr**
I didn't forget about

- [x] Tests (unnecessary)
- [x] Changelog
- [x] Documentation

Close #763

### CODE REVIEW

File: webui/src/components/ReplicasetList/ReplicasetList.js [hunk 1]
```diff
@@ -116,7 +116,36 @@ const styles = {
...
+  }
+
+  const a_leader = a.servers.find(server => server.uuid === a.master.uuid);
+  const b_leader = b.servers.find(server => server.uuid === b.master.uuid);
+
+  result = compareAliases(a_leader.alias, b_leader.alias)
+  if (result) {
+    return result;
+  }
+  return b.uuid < a.uuid ? 1 : -1;
```
Comment (printercu) [inline webui/src/components/ReplicasetList/ReplicasetList.js:147]: Logic of handling missing values is spread over 2 functions. What about smth like this?
```js
let a_value = a.alias || ''
let b_value = b.alias || ''
if (a_value === b_value) {
  a_value = a.servers... || ''
  b_value = ...
}
if (a_value === b_value) {
  a_value = a.uuid
  b_value = ...
}
return a_value < b_value ? -1 : 1
```

File: webui/src/components/ReplicasetList/ReplicasetList.js [hunk 2]
```diff
@@ -116,7 +116,36 @@ const styles = {
...
+}
+
+const prepareReplicasetList = dataSource => [...dataSource].sort((a, b) => {
+  let result = compareAliases(a.alias, b.alias)
+  if (result) {
+    return result;
+  }
+
+  const a_leader = a.servers.find(server => server.uuid === a.master.uuid);
+  const b_leader = b.servers.find(server => server.uuid === b.master.uuid);
```
[the remark this answers is not in the document]
Comment (rosik) [reply webui/src/components/ReplicasetList/ReplicasetList.js:141]: You're right. In stateful failover `master` field could be switched automatically. Guess it's better to use `a.servers[0]`

File: webui/src/components/ReplicasetList/ReplicasetList.js [hunk 3]
```diff
@@ -116,7 +116,25 @@ const styles = {
...
+    aValue = a.servers[0].alias || '';
+    bValue = b.servers[0].alias || '';
+  }
+
+  if (aValue === bValue) {
+    aValue = a.uuid;
+    bValue = b.uuid;
+  }
+
+  if (aValue === '' || bValue === '') {
```
Comment (printercu) [inline webui/src/components/ReplicasetList/ReplicasetList.js:133]: what the case is it for?

Comment (HustonMmmavr) [reply webui/src/components/ReplicasetList/ReplicasetList.js:133]: Replicasets with empty aliases (or empty server aliases) will be at the end.

**Pull request merged**